### ISSUE TITLE
A11y_code.visualstudio.com_DocsUserGuide_UserInterface_K4W: Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by Mohini78** [microsoft/vscode-docs#2790]
“[Check out Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com)
Windows Version: Windows10
Browser Version:
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
MAS Violated: MAS1.4.3
Other Information : AI Auto issue

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/docs/getstarted/userinterface](https://code.visualstudio.com/docs/getstarted/userinterface) OR
follow below steps:
1.Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com)
2.Navigate to "Docs" link.
3.From left navigation, select "Get Started".
4. Navigate to "User Interface" link.
5.Use Accessibility Insights​ tool and run it and observe the issues.

**Actual:**
Element has insufficient color contrast of 3.85 (foreground color: #707070, background color: #e3e3e3, font size: 9.0pt, font weight: normal)

This issue is observed with Chrome(using Accessibility Insights Tool)

**Expected:**
Elements must have sufficient color contrast.
Expected contrast ratio of 4.5:1
Ensure color contrast of at least 4.5:1 for small text or 3:1 for large text, even if text is part of an image. Large text has been defined in the requirements as 18pt or 14pt bold.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)
Ensure all text elements have sufficient color contrast between the text in the foreground and background color behind it.

**User Impact:**
Some people with low vision experience low contrast, meaning that there aren't very many bright or dark areas. Everything tends to appear about the same brightness, which makes it hard to distinguish outlines, borders, edges, and details. Text that is too close in luminance (brightness) to the background can be hard to read.

There are nearly three times more individuals with low vision than those with total blindness. One in twelve people cannot see the average full spectrum of colors - about 8% of men and 0.4% of women in the US. A person with low vision or color blindness is unable to distinguish text against a background without sufficient contrast.

**MAS Reference:**
MAS1.4.3 -
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6})

**Attachment for Reference:**
![A11y_code visualstudio com_DocsUserGuide_Accessibility_K4W_1](https://user-images.githubusercontent.com/51443731/59995325-741de600-9674-11e9-9593-5f31ddad550f.png)
![A11y_code visualstudio com_DocsUserGuide_Accessibility_K4W_2](https://user-images.githubusercontent.com/51443731/59995326-74b67c80-9674-11e9-9fb4-bc5537558a5f.png)

**Comment by octref** [comment]
The bg is `#ffffff`, not `#e3e3e3`. The contrast ratio is 4.95:1.

![image](https://user-images.githubusercontent.com/4033249/60835192-5aea5e80-a177-11e9-9f8d-d89f0b7fd3be.png)
